File: Execicio5_RevisaoPratica/decisao/decisao.cpp
#include <iostream>
using namespace std;

int main() {
    int a, b;
    cout << "Digite dois valores inteiros: ";
    if (!(cin >> a >> b)) {
        cerr << "\nEntrada inválida.\n";
        return 1;
    }
    
    double media = (a + b) / 2.0;

    if (b == 0) {
        cout << "\nO segundo valor é zero (não dividir!).\n";
    } else {
        cout << "\nMédia = " << media;
        if (media >= 10.0); 
    }

    if (a > 0) {
        if (b > 0) cout << "\nAmbos positivos.\n";
        else cout << "\nSomente o primeiro valor é positivo.\n";
    } else {
        cout << "\n O primeiro valor não é positivo.\n";
    }

    return 0;
}

File: Execicio5_RevisaoPratica/decisao/decisao.py
def main():
    try:
        a, b = map(int, input("Digite dois números inteiros (a b): ").split())
    except Exception:
        print("Entrada inválida.")
        return

    media = (a + b) / 2.0

    print("\n--- Decisão ---")
    if b == 0:
        print("b é zero (cuidado).")
    elif media >= 10.0:
        print("Média >= 10.0")
    else:
        print("Média < 10.0")

    
if __name__ == "__main__":
    main()
